QATCH/VisQAI/src/view/model_selection_dialog.py

```
@@ -0,0 +1,667 @@
+                        self.pinned_names[filename] = metadata['pinned_name']
+
+                models.append({
+                    'filename': filename,
+                    'filepath': filepath,
+                    'pinned_name': self.pinned_names.get(filename, metadata.get('name', filename)),
+                    'created': os.path.getctime(filepath),
+                    'size': os.path.getsize(filepath),
+                    'metadata': metadata,
+                    'sha': sha
+                })
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)